src/main/ingestion-queues/queue.ts

```diff
@@ -49,50 +49,79 @@ export async function startQueue(
     }
 
     try {
+        const redisQueue = startQueueRedis(server, piscina, mergedWorkerMethods)
         if (server.KAFKA_ENABLED) {
             return await startQueueKafka(server, piscina, mergedWorkerMethods)
-        } else {
-            return startQueueRedis(server, piscina, mergedWorkerMethods)
         }
+        return redisQueue
```

Comment:
Since we want to do operations on drain: https://github.com/PostHog/plugin-server/blob/master/src/main/pluginsServer.ts#L138

Maybe makes sense to return a list of queues?

Reply:
This is actually a valid comment, but oh damn it's gonna be quite a mess

Reply:
I opted for keeping `queue` what it was: the ingestion queue. This is important because while we can do things like:

```
for (const queue of queues) {
    queue.resume()
}
```

... we can't really do the same for `isPaused`. So `queue` remains what it was + we now have a `redisQueueForKafkaIngestion` which is the jobs queue and is only needed for kafka given on celery ingestion the plugin jobs and ingestion queues are the same.

Reply:
🙈 . Thanks!